core/include/traccc/seeding/doublet_finding_helper.hpp

```diff
@@ -45,45 +45,46 @@ bool doublet_finding_helper::isCompatible(
     const internal_spacepoint<spacepoint>& sp1,
     const internal_spacepoint<spacepoint>& sp2, const seedfinder_config& config,
     bool bottom) {
+    bool result = true;
     if (bottom) {
         scalar deltaR = sp1.radius() - sp2.radius();
         // if r-distance is too big, try next SP in bin
         if (deltaR > config.deltaRMax) {
-            return false;
+            result = false;
         }
         // if r-distance is too small, continue because bins are NOT r-sorted
         if (deltaR < config.deltaRMin) {
-            return false;
+            result = false;
         }
         scalar cotTheta = (sp1.z() - sp2.z()) / deltaR;
```

Comment:
I was just thinking that we might have an issue here. While before if deltaR were 0 we would never get here, now there's theoretically the possibility of doing a division by 0. But adding an additional 
`if ( deltaR == 0) { return 0; }`

is probably not that good as well.

Reply:
I just realized there's an easy fix, just multiply the other term instead of dividing this one.